AssemblyLoadDebugger2022/source.extension.vsixmanifest

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
+    <Metadata>
+        <Identity Id="34f00824-cb6a-4bc1-a27d-7375239cdd2f" Version="1.2" Language="en-US" Publisher="Mike Lorbetske" />
+        <DisplayName>Assembly Load Debugger</DisplayName>
```

Comment:
@mlorbetske this section is now missing the license as I couldn't get that playing nice with the shared code.
In the resources folder there is a link to the LICENSE file, but when I reference that in the manifest, I get a compile error